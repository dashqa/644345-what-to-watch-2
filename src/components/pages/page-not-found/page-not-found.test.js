import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from './page-not-found';

it(`Not found page correctly renders after relaunch`, () => {
  const tree = renderer.create(<PageNotFound/>).toJSON();
  expect(tree).toMatchSnapshot();
});
