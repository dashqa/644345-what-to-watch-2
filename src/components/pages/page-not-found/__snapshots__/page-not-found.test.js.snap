// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page correctly renders after relaunch 1`] = `
<h1
  style={
    Object {
      "margin": "50px 0",
      "textAlign": "center",
    }
  }
>
  404: Not Found
</h1>
`;
